crates/democracy/src/lib.rs

```diff
@@ -335,46 +270,37 @@ pub mod pallet {
             PublicPropCount::<T>::put(0 as PropIndex);
             ReferendumCount::<T>::put(0 as ReferendumIndex);
             LowestUnbaked::<T>::put(0 as ReferendumIndex);
-            StorageVersion::<T>::put(Releases::V1);
         }
     }
 
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
-        /// A motion has been proposed by a public account. \[proposal_index, deposit\]
-        Proposed(PropIndex, BalanceOf<T>),
-        /// A public proposal has been tabled for referendum vote. \[proposal_index, deposit,
-        /// depositors\]
-        Tabled(PropIndex, BalanceOf<T>, Vec<T::AccountId>),
-        /// A referendum has begun. \[ref_index, threshold\]
-        Started(ReferendumIndex, VoteThreshold),
-        /// A proposal has been fast tracked. \[ref_index\]
-        FastTrack(ReferendumIndex),
-        /// A referendum has been fast tracked. \[ref_index\]
-        FastTrackReferendum(ReferendumIndex),
-        /// A proposal has been approved by referendum. \[ref_index\]
-        Passed(ReferendumIndex),
-        /// A proposal has been rejected by referendum. \[ref_index\]
-        NotPassed(ReferendumIndex),
-        /// A referendum has been cancelled. \[ref_index\]
-        Cancelled(ReferendumIndex),
-        /// A proposal has been enacted. \[ref_index, result\]
-        Executed(ReferendumIndex, DispatchResult),
-        /// A proposal's preimage was noted, and the deposit taken. \[proposal_hash, who, deposit\]
-        PreimageNoted(T::Hash, T::AccountId, BalanceOf<T>),
-        /// A proposal preimage was removed and used (the deposit was returned).
-        /// \[proposal_hash, provider, deposit\]
-        PreimageUsed(T::Hash, T::AccountId, BalanceOf<T>),
-        /// A proposal could not be executed because its preimage was invalid.
-        /// \[proposal_hash, ref_index\]
-        PreimageInvalid(T::Hash, ReferendumIndex),
-        /// A proposal could not be executed because its preimage was missing.
-        /// \[proposal_hash, ref_index\]
-        PreimageMissing(T::Hash, ReferendumIndex),
-        /// A registered preimage was removed and the deposit collected by the reaper.
-        /// \[proposal_hash, provider, deposit, reaper\]
-        PreimageReaped(T::Hash, T::AccountId, BalanceOf<T>, T::AccountId),
+        /// A motion has been proposed by a public account.
+        Proposed {
+            proposal_index: PropIndex,
+            deposit: BalanceOf<T>,
+        },
+        /// A public proposal has been tabled for referendum vote.
+        Tabled {
+            proposal_index: PropIndex,
+            deposit: BalanceOf<T>,
```

Comment:
you mentioned the change from tuple-type to struct-type, but note also that one of the fields got removed here